server/methods/core/orders.js

```diff
@@ -213,6 +213,63 @@ export const methods = {
     });
   },
 
+
+  /**
+   * orders/cancelOrder
+   *
+   * @summary Start the cancel order process
+   * @param {Object} order - order object
+   * @param {Boolean} returnToStock - condition to return product to stock
+   * @return {Object} ret
+   */
+  "orders/cancelOrder": function (order, returnToStock) {
+    check(order, Object);
+    check(returnToStock, Boolean);
+
+    if (!Reaction.hasPermission("orders")) {
+      throw new Meteor.Error(403, "Access Denied");
+    }
+
+    if (!returnToStock) ordersInventoryAdjust(order._id);
+
+    let paymentMethod = orderCreditMethod(order).paymentMethod;
+    paymentMethod = Object.assign(paymentMethod, { amount: Number(paymentMethod.amount) });
+    const invoiceTotal = order.billing[0].invoice.total;
+    const shipment = order.shipping[0];
+    const itemIds = shipment.items.map((item) => {
+      return item._id;
+    });
+
+    // refund payment to customer
+    Meteor.call("orders/refunds/create", order._id, paymentMethod, Number(invoiceTotal), err => {
+      if (err) Logger.warn(err);
```

Comment:
Sometimes we're calling `Logger.warn` sometimes `Logger.error`. We should be consistent. Generally we should use `Logger.error` when handling an error, especially from a server-side Meteor method.